elevate-cpanel

```diff
@@ -443,6 +443,8 @@ sub start ($self) {
 
     }
 
+    $self->give_last_chance();
```

Comment:
I understand that this is fixing a bug but this is now displaying that `last_chance` warning so early that we should not display it when running only the checks. 
Previously it was only happening when attempting an elevation.
It seems that it's going to always occur

Reply:
sorry this is probably good as this is called from start and check is another code path